src/providers/Project/index.tsx

```diff
@@ -192,7 +195,8 @@ export const ProjectProvider: React.FC<ProjectProviderProps> = ({
     } catch (e) {
       console.error(e);
       setIsSaving(false);
-      showError(e?.message);
+      setApplicationErrorMessage(translateError(e?.message));
+      setShowErrorToast(true);
```

Comment:
Are there Application Error message that should not show in a toast?
If not should show error when `applicationErrorMessage` has a value. Prob don't need `showErrorToast`

Reply:
https://dapperlabs.slack.com/archives/C03UWGN8M0Q/p1670610305007239?thread_ts=1670609513.928649&cid=C03UWGN8M0Q error messaging will depend on what we discussed here

Reply:
not what I'm saying.
displaying an error toast is uses two properties. `applicationErrorMessage` and `showErrorToast`
Show not show a toast when ever applicationErrorMessage is set. Then you don't need `showErrorToast` property.